modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -1051,6 +1051,76 @@ definition {
 		}
 	}
 
+	@description = "LPS-164948 - Verify that the Label, Output and Formula Builder can be edited after the Custom Object be published"
+	@priority = 4
+	test CanEditFormulaBuilderOnCustomObject {
```

Comment:
Change the test name to `CanEditFormulaFieldOnCustomObject`

I updated the ticket. This way is better to identify.